server/src/test/java/org/elasticsearch/recovery/RecoveryWhileUnderLoadIT.java

```diff
@@ -260,7 +285,8 @@ public void testRecoverWhileRelocating() throws Exception {
         assertAcked(prepareCreate("test", 3, Settings.builder()
                 .put(SETTING_NUMBER_OF_SHARDS, numShards)
                 .put(SETTING_NUMBER_OF_REPLICAS, numReplicas)
-                .put(IndexSettings.INDEX_TRANSLOG_DURABILITY_SETTING.getKey(), Translog.Durability.ASYNC)));
+                .put(IndexSettings.INDEX_TRANSLOG_DURABILITY_SETTING.getKey(), Translog.Durability.ASYNC)
+                .put(IndexService.RETENTION_LEASE_SYNC_INTERVAL_SETTING.getKey(), randomFrom("100ms"))));
```

Comment:
There isn’t any randomization here? Inadvertently left over from testing, you meant to also include the default?

Reply:
ahahahaha. I wanted to run some extensive tests on the lowest value so I temporarily deleted the other ones :D fixed in ec75173

Reply:
That’s what I figured. :smile: